modules/visual_script/visual_script_property_selector.h

```diff
@@ -0,0 +1,92 @@
+/*************************************************************************/
+/*  property_selector.h                                                  */
```

Comment:
It should be `visual_script_property_selector.h`.

Reply:
Fixed.